pset-iterators.js

```diff
@@ -92,7 +106,12 @@ Tip: your new array should contain 45 elements
 console.log('Problem 3:')
 
 // Add your code below this line
+const presidentialParties = presidents.map(function (president) {
+  return president.party
+})
 
+console.log("All the parties the presidents belonged to:")
+console.log(presidentialParties)
```

Comment:
@AtleLohrmann Good Job here using the map method to return on the president party name.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94)